src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -67,38 +89,44 @@
   private static final String PRIVATE_MESSAGE_USER_SELECTOR = "#priv-user-%s";
   private static final String PRIVATE_MESSAGE_SELECTED_USER_SELECTOR =
       "#priv-user-%s.User.selected";
-  private static final String MESSAGE_ON_CHAT = "//span[@class='message'][contains(text(), '%s')]";
+  private static final String PATH_MESSAGE_ON_CHAT = "//span[@class='message'][contains(text(), '%s')]";
   private static final String NOTIFICATION_COUNTER =
       "//span[@class='splotch' and contains(text(), '%s')]";
+  private final String USER_NAME_ON_USER_PROFILE_SCREEN = userNameTitle.getText();
+  private final String USER_MESSAGE_ON_CHAT = String.format("Hello this is test message timestamp: ") + getTimeStamp();
 
-  public ChatPageObject(WebDriver driver) {
-    super();
+  private static final int REGULAR_USER_DROBDOWN_ELEMENTS = 3;
+
+  public ChatPage open(){
+    getUrl(urlBuilder.getUrlForWiki() + URLsContent.SPECIAL_CHAT);
+
+    return this;
   }
 
-  public void verifyChatPage() {
-    wait.forElementVisible(messageWritingArea);
-    wait.forElementVisible(chatInlineAlert);
-    wait.forElementVisible(sideBar);
-    wait.forElementVisible(userName);
-    wait.forElementVisible(userAvatar);
-    PageObjectLogging.log("verifyChatPage", "Chat page verified", true, driver);
+  public boolean isUserOnChat() {
+    try {
+      return chatInlineAlert.isDisplayed();
+    }catch (ElementNotFoundException e){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)